actions/send-sves.vbs

```diff
@@ -44,62 +44,60 @@ changelog = array()
 
 'INSERT ACTUAL CHANGES HERE, WITH PARAMETERS DATE, DESCRIPTION, AND SCRIPTWRITER. **ENSURE THE MOST RECENT CHANGE GOES ON TOP!!**
 'Example: call changelog_update("01/01/2000", "The script has been updated to fix a typo on the initial dialog.", "Jane Public, Oak County")
+call changelog_update("06/21/2022", "Updated handling for non-disclosure agreement and closing documentation.", "MiKayla Handley") '#493
 call changelog_update("02/22/2018", "Added option to send QURY for covered quarters.", "Ilse Ferris, Hennepin County")
 call changelog_update("11/28/2016", "Initial version.", "Charles Potter, DHS")
 
 'Actually displays the changelog. This function uses a text file located in the My Documents folder. It stores the name of the script file and a description of the most recent viewed change.
 changelog_display
 'END CHANGELOG BLOCK =======================================================================================================
 
-'THE SCRIPT----------------------------------------------------------------------------------------------------
-'Connects to BlueZone
+'THE SCRIPT-----------------------------------------------------------------------------------------------------------------
+'Connects to BLUEZONE
 EMConnect ""
-
 'Makes sure we're in MAXIS
 Call check_for_MAXIS(False)
 
-'Grabs case number
-call MAXIS_case_number_finder(MAXIS_case_number)
+'Grabs the MAXIS case number
+CALL MAXIS_case_number_finder(MAXIS_case_number)
+
 member_number = "01"
 
 'Shows and defines the initial dialog
 Dialog1 = ""
-BeginDialog Dialog1, 0, 0, 271, 105, "Send SVES Dialog"
-  EditBox 90, 5, 60, 15, MAXIS_case_number
-  EditBox 125, 25, 25, 15, member_number
-  CheckBox 5, 50, 165, 10, "Check here to case note that a QURY was sent.", case_note_checkbox
+BeginDialog Dialog1, 0, 0, 201, 120, "SEND SVES"
```

Comment:
Out-of-scope updates were made to this script. Create a new issue, site your sources, and move that code to a new branch. Submit only the changes to the INFC portion to this pull request.